diff --git a/setup.py b/setup.py
index 598a962..62121c3 100644
--- a/setup.py
+++ b/setup.py
@@ -126,10 +126,9 @@ setup(
         "six",
         # For pkg_resources
         "setuptools",
-        # version restrictions to support Python 2
-        "GitPython<2.1.12",
-        "python-gitlab<2",  # this is also needed to support Python 3.5
-        "gitdb2<3",
+        "GitPython",
+        "python-gitlab",
+        "gitdb2",
     ],
     # List additional groups of dependencies here (e.g. development
     # dependencies). Users will be able to install these using the "extras"
