diff --git a/setup.py b/setup.py
index 8e9b519..8fb36fb 100644
--- a/setup.py
+++ b/setup.py
@@ -122,10 +122,9 @@ setup(
         "yapf==0.24.0",
         "whichcraft",
         "six",
-        # version restrictions to support Python 2
-        "GitPython<2.1.12",
-        "python-gitlab<2",  # this is also needed to support Python 3.5
-        "gitdb2<3",
+        "GitPython",
+        "python-gitlab",
+        "gitdb2",
     ],
     # List additional groups of dependencies here (e.g. development
     # dependencies). Users will be able to install these using the "extras"
